commit 7629ebb853687afe144987d301d77a4ff99c65b5
Author: Mark A. Grondona <mgrondona@llnl.gov>
Date:   Mon Sep 28 17:49:15 2009 +0000

    Don't make missing dmidecode a fatal error.

diff --git a/src/util/edac-ctl.in b/src/util/edac-ctl.in
index 5bab397..e0c169e 100755
--- a/src/util/edac-ctl.in
+++ b/src/util/edac-ctl.in
@@ -30,7 +30,7 @@ use Getopt::Long;
 
 my $prefix      = "@prefix@";
 my $sysconfdir  = "@sysconfdir@";
-my $dmidecode   = find_prog ("dmidecode") or exit (1);
+my $dmidecode   = find_prog ("dmidecode");
 my $modprobe    = find_prog ("modprobe")  or exit (1);
 
 my %conf        = ();
@@ -392,7 +392,7 @@ sub find_prog
     for my $dir ("/sbin", "/usr/sbin", split ':', $ENV{PATH}) {
         return "$dir/$file" if -x "$dir/$file";
     }
-    log_error ("Failed to find $file in PATH\n");
+    # log_error ("Failed to find $file in PATH\n");
     return "";
 }
 
