---
 src/util/edac-ctl.in |    2 ++
 1 file changed, 2 insertions(+)

--- edac-utils-0.16.orig/src/util/edac-ctl.in	2015-10-05 09:50:44.216884955 -0400
+++ edac-utils-0.16/src/util/edac-ctl.in	2015-10-05 11:11:41.451961491 -0400
@@ -166,6 +166,8 @@ sub guess_vendor_model_dmidecode {
     my ($vendor, $model);
     my $line = 0;
 
+    return ('Unknown', 'Unknown') unless $dmidecode;
+
     $< == 0 || die "Must be root to run dmidecode\n";
 
     open (DMI, "$dmidecode |") or die "failed to run $dmidecode: $!\n";
