From e34919489fc5d7e67c018efd1ca50c7c2d8fc1ce Mon Sep 17 00:00:00 2001
From: Mauro Carvalho Chehab <mchehab@redhat.com>
Date: Mon, 4 Oct 2010 12:29:44 -0300
Subject: [PATCH 4/4] edac-util.1: Fix a few mistakes at the spec

Signed-off-by: Mauro Carvalho Chehab <mchehab@redhat.com>
---
 src/util/edac-util.1.in |    8 ++++----
 1 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/src/util/edac-util.1.in b/src/util/edac-util.1.in
index 72b97f4..418b0c2 100644
--- a/src/util/edac-util.1.in
+++ b/src/util/edac-util.1.in
@@ -83,8 +83,8 @@ labels have been registered in sysfs.
 
 The default report will also
 display any errors that do not have any DIMM information. These
-errors occur when errors are reported in the memory controllers
-overflow or similar registers, indicating that more than one error
+errors occur when errors are reported in the memory controller
+overflow register, indicating that more than one error
 occurred during a given EDAC poll cycle. It is usually obvious
 from which DIMM locations these errors were generated.
 .TP
@@ -114,8 +114,8 @@ With the \fI\-\-quiet\fR option, output will be suppressed unless
 there are 1 or more errors to report.
 .TP
 .B ce
-This report simply displays the total number of Uncorrected Errors
-(UEs) detected on the system.
+This report simply displays the total number of Corrected Errors
+(CEs) detected on the system.
 With the \fI\-\-quiet\fR option, output will be suppressed unless
 there are 1 or more errors to report.
 
-- 
1.7.1

